app/views/components/_important_metadata.html.erb

```diff
@@ -0,0 +1,21 @@
+<% add_app_component_stylesheet("important-metadata") %>
```

Comment:
This component was recently removed in https://github.com/alphagov/government-frontend/pull/3414 so you can drop this commit, and update the view